contracts/oss.yml

```diff
@@ -6531,187 +6539,6 @@ paths:
         default:
           description: Unexpected error
           $ref: '#/components/responses/GeneralServerError'
-  /legacy/authorizations:
```

Comment:
I'm concerned that consumers of `contracts/oss.yml` will be missing this route (still found in ref/oss)